libs/ui/src/components/MediaItem/type/ImageMedia.vue

```diff
@@ -20,6 +21,19 @@ defineProps<{
   original: boolean
   isDetail?: boolean
 }>()
+const { $consola } = useNuxtApp()
+const { isDarkMode } = useTheme()
+const placeholder = computed(() => {
+  return isDarkMode.value ? '/placeholder.webp' : '/placeholder-white.webp'
+})
+
+const onError = (e: Event) => {
+  const target = e.target as HTMLImageElement
+  if (target) {
+    $consola.log('[KODADOT::IMAGE] unable to load', e)
+    target.src = placeholder.value
+  }
+}
```

Comment:
I would move this at component level

Reply:
The error is throw directly from the img tag, so I guess placing an error handler on this component is not bad.